### PR TITLE
Documentation for LabGraph Installation on MacOS

### PR DESCRIPTION
**Pull request by celikbasak**
## Description

Installation documentation is added for macOS 12.x and 11.x. 
Readme is updated.

## Type of change

- [x] Documentation update. No changes have been made to the code itself.

### CODE REVIEW

File: docs/macos_documentation/installation.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# MacOS Installation Guide
```
Comment (jfResearchEng) [inline docs/macos_documentation/installation.md:1]: Thanks for the update! Can we use the following directory instead: docs/installation/macos.md ?

File: docs/macos_documentation/installation.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (jfResearchEng) [inline docs/macos_documentation/installation.md:2]: Can you help add something like:
If the following error appears, you can use the following instructions:

File: README.md [hunk 3]
```diff
@@ -14,6 +14,10 @@ LabGraph is a streaming framework built by the Facebook Reality Labs Research te
 ```
 pip install labgraph
 ```
+For Mac (supported version as of 02/02/2022):
+```
+pip install labgraph==2.0.0
```
Comment (jfResearchEng) [inline README.md:19]: Let's keep "pip install labgraph"

Can you add a note in README.md
For more documentation on installation trouble shooting, please use this [link](https://github.com/facebookresearch/labgraph/tree/main/docs/installation).

**Comment by fzchriha** [comment]
@celikbasak thank you so much for this new documentation, I have been struggling with setting up labgraph on my MacOS for two days now, and I kept facing this error
Screen_Shot_2022-02-01_at_7 57 15_PM
 
Thanks to your documentation I'm finally able to run labgraph on my personal machine 😊

File: docs/macos_documentation/installation.md (hunk 1 above)
Comment (celikbasak) [reply docs/macos_documentation/installation.md]: Thank you for the recommendation! I've updated the directory.

File: docs/macos_documentation/installation.md (hunk 2 above)
Comment (celikbasak) [reply docs/macos_documentation/installation.md]: I've updated the document and added the related error message.

**Review by jfResearchEng — APPROVED** [review]
Lgtm, thanks for the updates!

**Pull request merged**